src/main/kotlin/blackJack/domain/player/Player.kt

```diff
@@ -0,0 +1,34 @@
+package blackJack.domain.player
+
+import blackJack.domain.card.Cards
+import blackJack.domain.enums.Status
+import blackJack.domain.enums.Status.HIT
+import blackJack.error.ErrorMessage
+
+class Player(val name: String, cards: Cards = Cards(emptyList()), status: Status = HIT) : Participant(status, cards) {
```

Comment:
```suggestion
class Player(val name: String) : Participant(status, cards) {
```
위와 같이 Player의 파라미터인 cards와 status를 제거 해볼까요?

Reply:
좋은 의견이신것 같습니다 다만, 위처럼 변경하게 되면 status, cards에 넣은 값이 없어 아래와 같이 구현하였는데 의도하신게 맞으실까요??
`class Player(val name: String) : Participant() {`
`class Dealer(val name: String) : Participant()`
`open class Participant(var status: Status = Status.HIT, var cards: Cards = Cards(emptyList())) {`